code/C++/ramniwas.cpp

```diff
@@ -0,0 +1,7 @@
+#include<iostream>
+
+int main()
+{
+std::cout<<"Hello World";
+return 0;
+}
```

Comment:
- Add using namespace std;
- remove std
```suggestion
using namespace std;
int main()
{
cout <<"Hello World"  << endl;
return 0;
}
```